composer.json

```diff
@@ -9,17 +9,18 @@
         }
     ],
     "require": {
-        "php": ">=5.5",
-        "webmozart/console": "^1.0-beta5",
+        "php": "^7.0 || ^7.1",
```

Comment:
either `~7.0.0 || ~7.1.0` which is equivalent to `7.{0,1}.*` or `^7.0` which is equivalent to `7.*`